ignite/pkg/cosmoserror/error_test.go

```diff
@@ -0,0 +1,83 @@
+package cosmoserror_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/ignite/cli/ignite/pkg/cosmoserror"
+)
+
+func TestUnwrap(t *testing.T) {
+	tests := []struct {
+		name string
+		err  error
+		want error
+	}{
+		{
+			name: "should return internal error",
+			err:  status.Error(codes.Internal, "test error 1"),
```

Comment:
```suggestion
			err:  fmt.Errorf("oups: %w", status.Error(codes.Internal, "test error 1")),
```

This change causes the test to fail, do you think it's OK ?